.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,5 @@
+{
+    "image": "mosaicml/pytorch_vision:latest",
+    "remoteUser": "mosaicml",
```

Comment:
Is it possible to use `root` as is default in the image and just use a virtual environment?  I think we've gone back and forth on this enough, and it's better to keep it consistent across everything if possible.  I don't want developers to have to think about what they need to do differently if they want to run their code on the other images.